style.css

```diff
@@ -0,0 +1,832 @@
+}
+
+.about {
+  justify-content: center;
+}
+
+.about #sup_text {
+  background-color: white;
+  width: 90%;
+  margin-inline: 2.5%;
+  border: 1px solid rgb(177, 177, 177);
+  font-family: 'Lato', sans-serif;
+  padding: 10px;
+  margin-top: 50px;
+  line-height: 30px;
+}
+
+.about #suptext2 {
+  width: 60%;
+  margin-inline: 20%;
+}
+
+h3 {
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+  margin: 0;
+  font-weight: 100;
+}
+
+.about h3 {
+  align-self: center;
+}
+
+.about #sup_text_desk {
+  display: none;
+  background-color: white;
+  width: 90%;
+  margin-inline: 2.5%;
+  border: 1px solid rgb(177, 177, 177);
+  font-family: 'Lato', sans-serif;
+  padding: 10px;
+}
+
+a:hover {
+  color: #ec5242;
+}
+
+.about p {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  margin-bottom: 30px;
+}
+
+h2,
+h4 {
+  color: #424242;
+  font-family: 'Lato', sans-serif;
+  margin: 0;
+}
+
+.header h4 {
+  margin-bottom: 40px;
+}
+
+.header h2 {
+  font-weight: bolder;
+  font-size: 25px;
+}
+
+.main h1 {
+  background-image: url(Images/h1bg.jpg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 50px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-family: 'CocoGoose', sans-serif;
+  line-height: 45px;
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+
+.about h1 {
+  color: #ec5242;
+  font-size: 50px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-family: 'CocoGoose', sans-serif;
+  line-height: 48px;
+  margin-top: 20px;
+  margin-bottom: 0;
+  text-align: center;
+}
+
+#program {
+  background-image: url(Images/Abstract-Patterns-High-Quality-Gray-Background.webp);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding-bottom: 80px;
+  display: flex;
+  flex-direction: column;
+}
+
+.lft {
+  text-align: right;
+  color: #ec5242;
+  font-size: 18px;
+  font-weight: bold;
+  width: 40%;
+}
+
+.plist {
+  width: 90%;
+  margin-inline: 5%;
+  padding-left: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.plist #f {
+  background-image: url(Images/Icons/conjunto-de-botones-de-xbox.png);
+  background-repeat: no-repeat;
+  background-size: 10%;
+  background-position: top 50% left 1%;
+}
+
+.plist #s {
+  background-image: url(Images/Icons/control-de-juego-con-joystick.png);
+  background-repeat: no-repeat;
+  background-size: 10%;
+  background-position: top 50% left 1%;
+}
+
+.plist #t {
+  background-image: url(Images/Icons/objetivo-del-juego-de-disparos.png);
+  background-repeat: no-repeat;
+  background-size: 10%;
+  background-position: top 50% left 1%;
+}
+
+.plist #fr {
+  background-image: url(Images/Icons/disco-de-forma-circular-negra.png);
+  background-repeat: no-repeat;
+  background-size: 10%;
+  background-position: top 50% left 1%;
+}
+
+.plist #ff {
+  background-image: url(Images/Icons/supervisar-el-teclado-y-el-mouse.png);
+  background-repeat: no-repeat;
+  background-size: 10%;
+  background-position: top 50% left 1%;
+}
```

Comment:
- Kindly ensure this card has the sam size with the other cards 
- 
![cardd](https://user-images.githubusercontent.com/59397159/181861986-26582901-fc76-4776-a2d9-3bd45e2a7ad6.png)

Reply:
***README FILE***

- Please ensure you've deployed using gh pages
- Add the video presentation in the readme file
- Update the description as it's not referencing this project
- Add the link to the issues page in the contributing header text contents
- Add the live demo link
- Complete the setup on the git clone command by adding repo link